plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -442,47 +545,11 @@ def process(
         self.profiles_dir = self._get_profiles_dir()
         fname_absolute_path = os.path.abspath(fname) if fname != "stdin" else fname
 
+        # NOTE: dbt exceptions are caught and handled safely for pickling by the outer
+        # `handle_dbt_errors` decorator.
         try:
             os.chdir(self.project_dir)
-            processed_result = self._unsafe_process(fname_absolute_path, in_str, config)
-            # Reset the fail counter
-            self._sequential_fails = 0
```

Comment:
It looks like the logic for detecting multiple (sequential) failures was removed in this PR. Was that a deliberate change?

Reply:
Yes - it didn't have test coverage, and all of the times that it used to be used don't seem to apply anymore.

Reply:
Yeah, looking back at when those sequential fails were introduced, it was to solve #1088 (with PR #1106, which built on #1044). That related to the early days of parallel linting for dbt, which I think we now have more elegant and robust mechanisms for.